src/web/assets/js/main.js

```diff
@@ -1,23 +1,37 @@
 $( document ).ready(function() {
     console.log("Yeah, I work! Document is ready.");  // custom JS will be here
 
-    function log( message ) {
-        $( "<div class='message'>" ).text( message ).prependTo( "#log" );
-        $( "#log" ).scrollTop( 0 );
-    }
+	function log( message, log_type ) {
+		var log_div = '#log_' + log_type;
+		$( "<div class='message'>" ).text( message ).prependTo( log_div );
+		$( "#log" ).scrollTop( 0 );
+	}
 
-    var path = $("#modal_input").data("autocomplete-path")
-    $( "#modal_input" ).autocomplete({
-        source: path,
-        minLength: 2,
-        appendTo: "#log",
-        position: { my : "right tops", at: "right bottom" },
-        response: function( event, ui ) {
-            if(ui.content.length === 0) {
-                log( "No results found" )
-            } else {
-                $('.message').remove()
-            }
-        }
-    });
+	var path = $("#modal_input_assignOrganizer").data("autocomplete-path")
```

Comment:
fuj to ID je nechutne, tak bud s podtrznikmi ale camel case.

Reply:
Oki